ocs_ci/utility/rosa.py

```diff
@@ -709,3 +710,46 @@ def edit_addon_installation(
     utils.run_cmd(cmd)
     if wait:
         wait_for_addon_to_be_ready(cluster_name, addon_name)
+
+
+def configure_managed_service_size(size, cluster_name=None):
+    """
+    Configure the number of osds and instances
+
+    Args:
+        size (str): Cluster size
+        cluster_name (str): The cluster name. The default value is 'config.ENV_DATA["cluster_name"]'
+
+    """
+    cluster_name = cluster_name or config.ENV_DATA["cluster_name"]
+    service_id = get_rosa_cluster_service_id(cluster_name)
+    cmd = f"rosa edit service --id={service_id} --size {size}"
+    utils.run_cmd(cmd)
+
+
+def get_managed_service_size(cluster_name=None):
+    """
+    Configure the number of osds and instances
+
+    Args:
+        cluster_name (str): The cluster name. The default value is 'config.ENV_DATA["cluster_name"]'
+
+    Returns:
+        int: return current size
+
+    """
+    cluster_name = cluster_name or config.ENV_DATA["cluster_name"]
+    cmd = "rosa list service"
+    cmd_output = utils.run_cmd(cmd)
+    time.sleep(1)
+    lines = cmd_output.splitlines()
+    for line in lines:
+        if cluster_name in line:
+            addon = line.split()[1]
+    cmd = f"rosa describe addon-installation -c {cluster_name} --addon {addon}"
+    cmd_output = utils.run_cmd(cmd)
+    lines = cmd_output.splitlines()
+    for line in lines:
+        if "size" in line:
```

Comment:
isn't there a more proper way of fetching the size?

Reply:
discussed offline and @OdedViner will check if we can get it in a certain format, such as json, in order to fetch the size value properly

Reply:
```
$ rosa describe addon-installation -c jijoy8-14-pr --addon ocs-provider-qe -o json
Error: unknown shorthand flag: 'o' in -o
Usage:
  rosa describe addon-installation clusterID AddonInstallationID [flags]

Aliases:
  addon-installation, add-on-installation

Examples:
  # Describe the 'bar' add-on installation on cluster 'foo'
  rosa describe addon-installation --cluster foo --addon bar

Flags:
      --addon string     Name or ID of the addon installation (required).
  -c, --cluster string   Name or ID of the cluster to list the add-ons of (required).
  -h, --help             help for addon-installation

Global Flags:
      --color string     Surround certain characters with escape sequences to display them in color on the terminal. Allowed options are [auto never always] (default "auto")
      --debug            Enable debug mode.
      --profile string   Use a specific AWS profile from your credential file.
      --region string    Use a specific AWS region, overriding the AWS_REGION environment variable.

Failed to execute root command: unknown shorthand flag: 'o' in -o
```

```
$ rosa describe addon-installation -c jijoy8-14-pr --addon ocs-provider-qe --output json
Error: unknown flag: --output
```